SCF-Session-8/zooManagement/Bird.java

```diff
@@ -0,0 +1,46 @@
+package zooManagement;
+
+/**
+ * This Bird class inherits the Animal super class
+ * @author Ishita_Mathur
+ */
+public class Bird extends Animal
+{
+	private final boolean isAbleToFly;
+	private double wingsSpan;
+	
+	/**
+	 * The constructor initializes the Birds in the Zoo
+	 * It calls its superclass constructor to set it into animal property 
+	 * @param name
+	 * @param weight
+	 * @param age
+	 * @param sound
+	 * @param numberOfLegs
+	 * @param isAbleToFly
+	 * @param wingsSpan
+	 * @param animalType
+	 */
+	public Bird(String name, double weight, int age, String sound, int numberOfLegs, boolean isAbleToFly, double wingsSpan, AnimalName animalType)
+	{	
```

Comment:
Parameters should be in a sequence like first all for super class and then for the current class.